core/src/main/java/org/apache/accumulo/fate/AdminUtil.java

```diff
@@ -478,32 +411,32 @@ public boolean prepFail(TStore<T> zs, ZooReaderWriter zk, ServiceLockPath zLockM
     try {
       txid = Long.parseLong(txidStr, 16);
     } catch (NumberFormatException nfe) {
-      System.out.printf("Invalid transaction ID format: %s%n", txidStr);
+      log.error("Invalid transaction ID format: {}", txidStr);
       return false;
     }
     boolean state = false;
     zs.reserve(txid);
     TStatus ts = zs.getStatus(txid);
     switch (ts) {
       case UNKNOWN:
-        System.out.printf("Invalid transaction ID: %016x%n", txid);
+        log.error("Invalid transaction ID: {}", txid);
```

Comment:
These are a bit of a change in behavior for the shell, which I think currently uses AdminUtil. I think this behavior is better, but might want to include some release notes that mention the change, so if anybody was relying on scripting the shell, and capturing STDOUT, it will work differently for them, depending on their log configuration.